devel/zanata-cli/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                zanata-cli
+version             4.4.5
+categories          devel
+platforms           darwin
+license             LGPL-2.1+
+maintainers         {l2dy @l2dy} openmaintainer
+description         Zanata Java command-line client
+long_description    ${description}
+
+homepage            https://github.com/zanata/zanata-platform
+master_sites        https://search.maven.org/remotecontent?filepath=org/zanata/zanata-cli/${version}
+
+extract.suffix      -dist.tar.gz
+
+checksums           rmd160  39638ec3f440249b001eeb7de8d3cbc318ae7c27 \
+                    sha256  9f7823b64c8feff17fb42a3925dba58563b102028ae883d104a89682106f95ba \
+                    size    16149108
+
+depends_run         bin:java:kaffe
+
+use_configure       no
+
+build {}
```

Comment:
Should this just be `build`? I can't remember, but I seem to recall that there's some such convention @ryandesign had been implementing across the Portfiles.

Reply:
I don't think so, currently 227 ports have `^build {}$` and no port have `^build$`.

Reply:
I'm going to leave this be then, and if @ryandesign thinks it should be done differently I can fix it later or you can.